src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -368,16 +382,20 @@ hours, so the inferred interval is 1 day (daily), ``P1D``.
 If the limit is omitted, unlimited cycling is assumed. This will be
 bounded by the workflow's final cycle point if given.
 
-Another supported form of ISO8601 :term:`recurrence` is:
-``R[limit?]/[interval]/[datetime]``. This form uses the
-datetime as the end of the cycling sequence rather than the start.
-For example, ``R3/P5D/20140430T06`` means:
+
+Format 4: ``R[limit?]/[interval]/[datetime]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Another supported recurrence form is:
+``R[limit?]/[interval]/[datetime]`` (format number 4 in the ISO 8601 standard).
```

Comment:
A second reason why the section order and the format number don't match to add to the note at the top of the Format3 section?